### PR TITLE
FIX: Typo in documentation example

### LINKED ISSUE #1807: Documentation bug
### Describe the bug

I believe there is a bracket out of place, see below...

### To Reproduce

1. Go to https://stellargraph.readthedocs.io/en/stable/api.html
2. CTRL-F "Heterogeneous graphs, with multiple node o" to jump to cope snippet

### Observed behavior

The code snippet has a misplaced round bracket. Running the below code snippet results in the following error:

```
>>> foo_nodes = IndexedArray(np.array([[-1]]), index=["a"])
>>> bar_nodes = IndexedArray(
>>>    np.array([[0.4, 100], [0.1, 200], [0.9, 300]], index=["b", "c", "d"])
>>> )
>>>
>>> StellarGraph({"foo": foo_nodes, "bar": bar_nodes}, edges)

TypeError: 'index' is an invalid keyword argument for array()
```
### Expected behavior

```
>>> foo_nodes = IndexedArray(np.array([[-1]]), index=["a"])
>>> bar_nodes = IndexedArray(
>>>     np.array([[0.4, 100], [0.1, 200], [0.9, 300]]), index=["b", "c", "d"]
>>> )
>>> 
>>> StellarGraph({"foo": foo_nodes, "bar": bar_nodes}, edges)
<stellargraph.core.graph.StellarGraph at 0x7f7bf07f14e0>
```

### Environment

```
Operating system: `Linux-4.15.0-117-generic-x86_64-with-debian-buster-sid`
```
Python version:
```
3.7.2 (default, Jul  3 2019, 11:00:30) 
[GCC 7.4.0]
```

Package versions:

<details>

```
absl-py==0.10.0
appdirs==1.4.4
astunparse==1.6.3
atpublic==2.0
backcall==0.2.0
black==20.8b1
bleach==3.2.0
blis==0.4.1
boto==2.49.0
boto3==1.14.63
botocore==1.17.63
cachetools==4.1.1
certifi==2020.6.20
cffi==1.14.2
chardet==3.0.4
click==7.1.2
colorama==0.4.3
commonmark==0.9.1
configobj==5.0.6
cryptography==3.1
cycler==0.10.0
cymem==2.0.3
Cython==0.29.21
decorator==4.4.2
dictdiffer==0.8.1
distro==1.5.0
docutils==0.15.2
dpath==2.0.1
dvc==1.7.2
Faker==4.1.3
flake8==3.8.3
flatten-dict==0.3.0
flufl.lock==3.2
ftfy==5.8
funcy==1.14
future==0.18.2
gast==0.3.3
gensim==3.8.3
gitdb==4.0.5
GitPython==3.1.8
google-auth==1.21.2
google-auth-oauthlib==0.4.1
google-pasta==0.2.0
grandalf==0.6
-e git+git@github.com:wellcometrust/grant-embedding.git@5b9cbe3e287141a756f262842cb94e4ef7f7bb4d#egg=grant_embeddings
grpcio==1.32.0
h5py==2.10.0
hdbscan==0.8.26
idna==2.10
importlib-metadata==1.7.0
ipdb==0.13.3
ipython==7.18.1
ipython-genutils==0.2.0
jedi==0.17.2
jeepney==0.4.3
jmespath==0.10.0
joblib==0.16.0
jsonpath-ng==1.5.2
Keras-Preprocessing==1.1.2
keyring==21.4.0
kiwisolver==1.2.0
llvmlite==0.34.0
Markdown==3.2.2
matplotlib==3.3.2
mccabe==0.6.1
murmurhash==1.0.2
mypy-extensions==0.4.3
nanotime==0.5.2
nervaluate==0.1.7
networkx==2.5
numba==0.51.2
numpy==1.19.2
oauthlib==3.1.0
opt-einsum==3.3.0
packaging==20.4
pandas==1.1.2
parso==0.7.1
pathlib2==2.3.5
pathspec==0.8.0
pexpect==4.8.0
pickleshare==0.7.5
Pillow==7.2.0
pkginfo==1.5.0.1
plac==0.9.6
ply==3.11
preshed==3.0.2
prompt-toolkit==3.0.7
protobuf==3.13.0
ptyprocess==0.6.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycodestyle==2.6.0
pycparser==2.20
pydot==1.4.1
pyflakes==2.2.0
Pygments==2.7.0
pygtrie==2.3.2
pyparsing==2.4.7
python-dateutil==2.8.1
pytz==2020.1
PyYAML==5.3.1
readme-renderer==26.0
regex==2020.7.14
requests==2.24.0
requests-oauthlib==1.3.0
requests-toolbelt==0.9.1
rfc3986==1.4.0
rich==6.2.0
rsa==4.6
ruamel.yaml==0.16.12
ruamel.yaml.clib==0.2.2
s3transfer==0.3.3
sacremoses==0.0.43
scikit-learn==0.23.2
scipy==1.5.2
SecretStorage==3.1.2
sentencepiece==0.1.91
shortuuid==1.0.1
shtab==1.3.1
six==1.15.0
smart-open==2.1.1
smmap==3.0.4
spacy==2.2.1
spacy-transformers==0.5.1
srsly==1.0.2
stellargraph==1.2.1
tabulate==0.8.7
tensorboard==2.3.0
tensorboard-plugin-wit==1.7.0
tensorflow==2.3.0
tensorflow-estimator==2.3.0
termcolor==1.1.0
text-unidecode==1.3
thinc==7.1.1
threadpoolctl==2.1.0
toml==0.10.1
torch==1.6.0
torchcontrib==0.0.2
tqdm==4.49.0
traitlets==5.0.4
transformers==2.0.0
twine==3.2.0
typed-ast==1.4.1
typer==0.3.2
typing-extensions==3.7.4.3
umap-learn==0.4.6
urllib3==1.25.10
voluptuous==0.11.7
wasabi==0.8.0
wcwidth==0.2.5
webencodings==0.5.1
wellcomeml==2020.9.0
Werkzeug==1.0.1
wrapt==1.12.1
zc.lockfile==2.0
zipp==3.1.0
```

</details>

### PR DESCRIPTION
**Pull request by thatlittleboy**
Small PR to resolve #1807

### CODE REVIEW

**Comment by thatlittleboy** [comment]
Currently blocked by upstream bug in black.

**Review by huonw — APPROVED** [review]
Thank you! It looks like CI should be passing (the most recent run failed due to a timeout), so I've retried it and will merge after it passes.

**Pull request merged**
